lidar1:
  roll: 1.5708       # 90度
  pitch: -1.5708     # -90度
  yaw: 1.5708        # 90度
  tx: 0.11
  ty: 0.0
  tz: 0.0

lidar2:
  roll: 0.0
  pitch: -1.5708
  yaw: 0.0
  tx: 0.0
  ty: 0.0
  tz: 0.0

